src/MSBuild/CommandLineSwitches.cs

```diff
@@ -394,6 +396,8 @@ private struct DetectedParameterizedSwitch
         private DetectedParameterizedSwitch[] _parameterizedSwitches;
         // NOTE: the above arrays are instance members because this class is not required to be a singleton
 
+        internal static Dictionary<string, string> SwitchesFromResponseFiles = new();
```

Comment:
Does this need to be order-preserving? Should it be a `List<(string Path, string Contents)>`?